code/modules/psionics/faculties/coercion.dm

```diff
@@ -300,7 +300,7 @@
 			if(!T || L == user || L.stat == DEAD || L.invisibility == INVISIBILITY_LEVEL_TWO)
 				continue
 			if(L.isSynthetic() || L.is_diona() || isvaurca(L))
-				var/obj/item/organ/internal/augment/psiaug = locate() in L
+				var/obj/item/organ/internal/augment/psi/psiaug = locate() in L
```

Comment:
```suggestion
				psiaug = has_psi_aug()
```

Or possibly use is_psi_blocked()

Reply:
travis shit the bed on that

Reply:
Yes, you just need to call the proc on the mob instead.

Reply:
has_psi_aug() returns boolean anyway